hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestFSImage.java

```diff
@@ -275,6 +276,31 @@ public void testSaveAndLoadStripedINodeFile() throws IOException{
     }
   }
 
+  @Test
+  public void testImportCheckpoint() throws IOException{
```

Comment:
A little advice. You can consider using try with resources for `MiniDFSCluster cluster = new MiniDFSCluster.Builder(config).build()`.

Reply:
> 

Thanks @tomscut for your advice , it seems simpler code.